### PR TITLE
6928 jobs errors

### PR DESCRIPTION
**Pull request by hooverlunch**

### CODE REVIEW

**Comment by emitche** [comment]
@linesarefuzzy You are the best person to review this issue.

**Comment by emitche** [comment]
@arisah-ijeoma or @linesarefuzzy If either of you has capacity to review this code, please do. Thank you!

File: .env.example [hunk 1]
```diff
@@ -7,6 +7,6 @@ export MADELINE_SMTP_PORT=587
 export MADELINE_SMTP_DOMAIN=example.com
 export MADELINE_SMTP_USERNAME=username
 export MADELINE_SMTP_PASSWORD=xxx
-
+export MADELINE_ERROR_EMAILS_TO=madeline.ops@sassafras.coop
```
Comment (emitche) [inline .env.example]: Should this be committed?

Comment (hooverlunch) [reply .env.example]: Good point. I'll change it to something generic.

Comment (emitche) [reply .env.example]: Thanks.

File: spec/jobs/recalculate_loan_health_job_spec.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    end
+  end
+
+  # Loan could get deleted between when job is enqueued and when it is run.
+  context "for non-existent loan" do
+    it "should do nothing and not raise error" do
+      described_class.new.perform(loan_id: 1234567)
+    end
+  end
+end
```
Comment (linesarefuzzy) [inline spec/jobs/recalculate_loan_health_job_spec.rb]: Great job on these tests. Thanks!

File: config/initializers/delayed_job.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# ActiveJob seems to provide no way to control this on a per-job basis.
+# Defining a max_attempts method does not seem to work.
+# Also, setting rescue_from seems to also immediately delete a job, so even if this were not set to 1,
+# retries do not appear to work in conjunction with rescue_from.
+# So we're disabling retry functionality for now because we don't need it for existing jobs.
+# We can use https://github.com/isaacseymour/activejob-retry or similar if we ever need
+# the ability to retry a job.
+Delayed::Worker.max_attempts = 1
```
Comment (linesarefuzzy) [inline config/initializers/delayed_job.rb]: Thanks for the comment.

File: app/jobs/recalculate_loan_health_job.rb [hunk 4]
```diff
@@ -1,12 +1,10 @@
...
-  def perform(loan_id: )
+class RecalculateLoanHealthJob < ApplicationJob
+  def perform(loan_id:)
     check = LoanHealthCheck.where(loan_id: loan_id).first
 
-    unless check
-      Rails.logger.debug "Creating health check for loan #{loan_id}"
-      check = LoanHealthCheck.create(loan_id: loan_id)
+    if check.nil?
+      return unless Loan.where(id: loan_id).exists?
```
[the remark this answers is not in the document]
Comment (hooverlunch) [reply app/jobs/recalculate_loan_health_job.rb]: We didn't. But it could happen if the job is scheduled and then the loan is deleted before it's run.

**Pull request merged**